mitmproxy/net/tls.py

```diff
@@ -21,29 +21,21 @@
 
 # redeclared here for strict type checking
 class Method(Enum):
-    # TODO: just SSL attributes once https://github.com/pyca/pyopenssl/pull/985 has landed.
-    TLS_SERVER_METHOD = getattr(SSL, "TLS_SERVER_METHOD", 8)
-    TLS_CLIENT_METHOD = getattr(SSL, "TLS_CLIENT_METHOD", 9)
+    TLS_SERVER_METHOD = getattr(SSL, "TLS_SERVER_METHOD")
+    TLS_CLIENT_METHOD = getattr(SSL, "TLS_CLIENT_METHOD")
```

Comment:
We don't need to use `getattr` here or below, we can just use `SSL.TLS_SERVER_METHOD` directly. :)